TypeCobol.LanguageServer.Test/LSRTests/ReplacingSyntaxOff/input/replacingSyntaxOff.tlsp

```diff
@@ -15,11 +15,11 @@
     },
     {
       "category": 1,
-      "message": "{\"jsonrpc\":\"2.0\",\"method\":\"textDocument/publishDiagnostics\",\"params\":{\"uri\":\"file:///C:/Users/BALLMA/AppData/Local/Temp/1/tcbl/TCOZBMX02229180100554248398.cee\",\"diagnostics\":[{\"range\":{\"start\":{\"line\":11,\"character\":23},\"end\":{\"line\":11,\"character\":41}},\"severity\":1,\"code\":\"27\",\"source\":\"Find the syntax diagram describing the statement in error in the language reference\",\"message\":\"Syntax error : Ambiguous reference to symbol CONVEA-TESTVEA-TEST \\r\\nSymbols found: DVZS0OSM::CONVEA::CONVEA-TESTVEA-TEST | DVZS0OSM::CONVEAL::CONVEA-TESTVEA-TEST\"},{\"range\":{\"start\":{\"line\":12,\"character\":23},\"end\":{\"line\":12,\"character\":42}},\"severity\":1,\"code\":\"27\",\"source\":\"Find the syntax diagram describing the statement in error in the language reference\",\"message\":\"Syntax error : Symbol CONVEAL-TESTVEA-TEST is not referenced\"}]}}"
+      "message": "{\"jsonrpc\":\"2.0\",\"method\":\"textDocument/publishDiagnostics\",\"params\":{\"uri\":\"file:///C:/Users/BALLMA/AppData/Local/Temp/1/tcbl/TCOZBMX02229180100554248398.cee\",\"diagnostics\":[{\"range\":{\"start\":{\"line\":11,\"character\":23},\"end\":{\"line\":11,\"character\":41}},\"severity\":1,\"code\":\"27\",\"source\":\"Find the syntax diagram describing the statement in error in the language reference\",\"message\":\"Syntax error : Ambiguous reference to symbol CONVEA-TESTVEA-TEST \\r\\nSymbols found: DVZS0OSM::CONVEA::CONVEA-TESTVEA-TEST | DVZS0OSM::CONVEAL::CONVEA-TESTVEA-TEST\"},{\"range\":{\"start\":{\"line\":12,\"character\":23},\"end\":{\"line\":12,\"character\":42}},\"severity\":1,\"code\":\"30\",\"source\":\"Page\",\"message\":\"Semantic error: Symbol CONVEAL-TESTVEA-TEST is not referenced\"}]}}"
```

Comment:
Same remark here : ambiguous reference to a symbol sounds more like a Semantic error than a syntax error.

Reply:
done.
https://github.com/TypeCobolTeam/TypeCobol/pull/1669/commits/4b5e1d0af58323248efbd75657d4b554eb7db081